### DISCUSSION TITLE
Sync ICS - Internet subscription calendar URLs

### DISCUSSION THREAD

**Discussion by phw198** [phw198/OutlookGoogleCalendarSync#1893] [Polls]
Outlook and Google make it possible to publish a calendar to an "ICS feed", which makes it available for others to subscribe to. This is a one-way, read only feed, but it does provide another potential option as a sync source.

This may be especially helpful if you cannot use the Outlook client or you [cannot get approval for OGCS](https://github.com/phw198/OutlookGoogleCalendarSync/discussions/1889) to access your Outlook Online account.

Note: Often subscribers like Google will only retrieve and update content from an ICS feed once every 24-48 hours. However, OGCS would be able to do it on your choice of frequency - the same way sync intervals can currently be configured!

### How to publish a calendar
* [Outlook instructions](https://support.microsoft.com/en-gb/office/share-your-calendar-in-outlook-com-0fc1cb48-569d-4d1e-ac20-5a9b3f5e6ff2) - see "Publish your calendar"
* [Google instructions](https://wafflebytes.blogspot.com/2019/02/google-calendar-how-to-share-your.html)

**Comment by tjeerdhans** [comment]
Unfortunately, this wouldn't work for me because I can only publish busy-free-information :-(

![image](https://github.com/user-attachments/assets/7a12e9ff-2aa9-46b8-b709-ffa3ac22cd34)

**Comment by phw198** [comment]
🔗 #1672, #2137
